app/services/comment_service.rb

```diff
@@ -43,6 +43,8 @@ def self.create(comment:, actor:, mark_as_read: true)
 
   def self.destroy(comment:, actor:)
     actor.ability.authorize!(:destroy, comment)
+    # paranoid destroy_all because comment votes seem to be dangling around.
+    comment.comment_votes.destroy_all
```

Comment:
actually .. because comment_votes have events which have notifications.. it
might be a bit involved to delete_all manually for everything concerned.

On Wed, 29 Apr 2015 at 19:07 Robert Guthrie <rguthrie@gmail.com> wrote:

> yea.. good idea..
>
> On Wed, 29 Apr 2015 at 18:52 James Kiesel <notifications@github.com>
> wrote:
>
>> In app/services/comment_service.rb
>> <https://github.com/loomio/loomio/pull/2127#discussion_r29313506>:
>>
>> > @@ -43,6 +43,8 @@ def self.create(comment:, actor:, mark_as_read: true)
>> >
>> >    def self.destroy(comment:, actor:)
>> >      actor.ability.authorize!(:destroy, comment)
>> > +    # paranoid destroy_all because comment votes seem to be dangling around.
>> > +    comment.comment_votes.destroy_all
>>
>> Can we do delete_all so it doesn't potentially take forever?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/loomio/loomio/pull/2127/files#r29313506>.
>>
>